src/daemon/input_linux.c

```diff
@@ -53,6 +53,10 @@ int uinputopen(struct uinput_user_dev* indev, int mouse){
 ///
 /// Some tips on using [uinput_user_dev in](http://thiemonge.org/getting-started-with-uinput)
 int os_inputopen(usbdevice* kb){
+    if(IS_SINGLE_EP(kb)) {
+        kb->uinput_kb = 0;
```

Comment:
Setting uinput_kb to 0 makes IS_CONNECTED fail, resulting in _updateconnected() not adding the device to ckb0/connected, and other issues.

Reply:
But if I don't set this, then os_inputclose might reference an ill-defined uinput_kb